x/liquidity/spec/03_state_transitions.md

```diff
@@ -1,174 +1,234 @@
-<!--
-order: 3
--->
+<!-- order: 3 -->
 
-# State Transitions
+ # State Transitions
+
+Messages (Msg) trigger state transitions.
 
 ## Coin Escrow for Liquidity Module Messages
 
-Three messages on the liquidity module need prior coin escrow before confirmation, which causes state transition on `Bank` module. Below lists are describing coin escrow processes for each given message type.
+Transaction confirmation causes state transition on `Bank` module. Some messages on the Liquidity Module require coin escrow before confirmation.
+
+The coin escrow processes for each message type are:
 
 ### MsgDepositWithinBatch
 
-To deposit coins into existing `Pool`, the depositor needs to escrow `DepositCoins` into `LiquidityModuleEscrowAccount`.
+To deposit coins into existing `Pool`, the depositor must escrow `DepositCoins` into `LiquidityModuleEscrowAccount`.
 
 ### MsgWithdrawWithinBatch
 
-To withdraw coins from `Pool`, the withdrawer needs to escrow `PoolCoin` into `LiquidityModuleEscrowAccount`.
+To withdraw coins from `Pool`, the withdrawer must escrow `PoolCoin` into `LiquidityModuleEscrowAccount`.
 
 ### MsgSwapWithinBatch
 
-To request coin swap, swap requestor needs to escrow `OfferCoin` into `LiquidityModuleEscrowAccount`.
+To request a coin swap, swap requestor must escrow `OfferCoin` into `LiquidityModuleEscrowAccount`.
 
 ## LiquidityPoolBatch Execution
 
-Batch execution causes state transitions on `Bank` module. Below categories describes state transition executed by each process in `PoolBatch` execution.
+Batch execution causes state transitions on the `Bank` module. The following categories describe state transition executed by each process in the `PoolBatch` execution.
```

Comment:
so we're updating the bank module?  we should link to https://docs.cosmos.network/master/modules/bank/

Reply:
Providing a link will certainly help. We expect devs who are familiar with Cosmos SDK so they must know bank module prior to use the liquidity module.